tests/integration/expected_test_results/product_read/get-existing-product.html

```diff
@@ -25,7 +25,7 @@
 <meta name="theme-color" content="#ffffff">
 
 	<meta name="apple-itunes-app" content="app-id=588797948">
-    <link rel="canonical" href="//world.openfoodfacts.localhost/product/0200000000034/some-product">
+    <link rel="canonical" href="//world.openfoodfacts.localhost/product/2000000000034/some-product">
```

Comment:
that one is strange, i can't reproduce it locally

Reply:
It's me that change the code, the test code was a 12 digits code, and now I issue a redirect for these.